package.json

```diff
@@ -81,6 +82,7 @@
     "bootstrap": "^4.2.1",
     "core-js": "^2.5.7",
     "hamburgers": "^1.1.3",
+    "i18n-lint": "jwarby/i18n-lint",
```

Comment:
please move dependency to devDependencies

Reply:
Moved to devDependencies